### PR TITLE
Adding relativeDigitMargin and absoluteDigitMargin properties to JDFlipNumberView

### PR DESCRIPTION
**Pull request by johnboiles**

### CODE REVIEW

File: JDFlipNumberView/JDFlipNumberView.m [hunk 1]
```diff
@@ -12,7 +12,6 @@
 
 
 static CGFloat JDFlipAnimationMinimumTimeInterval = 0.01; // = 100 fps
-static CGFloat JDFlipViewRelativeMargin = 0.05; // use 5% of width as margin
```
Comment (jaydee3) [inline JDFlipNumberView/JDFlipNumberView.m]: please keep this and set it as the default value. You can rename it to
`JDFlipViewDefaultRelativeMargin`.

File: JDFlipNumberView/JDFlipNumberView.m [hunk 2]
```diff
@@ -445,7 +445,7 @@ - (void)stopAnimation;
 - (NSUInteger)marginForWidth:(CGFloat)width;
 {
     if (self.digitViews.count <= 1) return 0;
-    return ((width*JDFlipViewRelativeMargin)/(self.digitViews.count-1));
+    return MAX(((width*self.relativeDigitMargin)/(self.digitViews.count-1)), self.absoluteDigitMargin);
```
Comment (jaydee3) [inline JDFlipNumberView/JDFlipNumberView.m]: Can we take the relativeMargin calculation in a seperate line and then just have a very readable MAX(relative,absolute)?

**Comment by jaydee3** [comment]
Thanks for doing this!

**Comment by johnboiles** [comment]
Done

**Comment by jaydee3** [comment]
awesome thanks a lot!

**Comment by johnboiles** [comment]
No problem! Thanks for JDFlipNumberView :)

**Pull request merged**